test/mergesort_test.js

```diff
@@ -0,0 +1,25 @@
+import { expect } from 'chai'
+import mergeSort from '../src/mergeSort.js'
+
+describe('mergeSort()', function(){
```

Comment:
Consider adding tests for empty arrays, and arrays that are already in sorted order, to make sure you're covering all possible cases.  You should also probably have tests for arrays with multiple types?